pkg/osdctlConfig/osdctlConfig.go

```diff
@@ -40,7 +30,8 @@ func EnsureConfigFile() error {
 
 	if err := viper.ReadInConfig(); err != nil {
 		return err
+	} else {
+		fmt.Println("Reading config file from " + configFilePath)
```

Comment:
```suggestion
		fmt.Println("Reading config file from " + configFilePath)
```

Reply:
The else isn't needed here. Also, imo this should be put behind a verbose flag or something. Many of the osdctl commands have json or other formatted output, and this output may interfere with piping to jq for example